tests/e2e/test_rosacli_idp.go

```diff
@@ -330,6 +330,7 @@ var _ = Describe("Edit IDP",
 					"-y")
 				Expect(err).To(BeNil())
 				textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+				Expect(textData).Should(ContainSubstring("Configuring IDP for cluster '%s'", clusterID))
```

Comment:
I do not think it is an excellent choice to validate the general message at each checkpoint. We should validate this in one checkpoint, and the others will do the distinguishing validation.

Reply:
@yasun1  it is neccessary to validate this step `Expect(textData).Should(ContainSubstring("Configuring IDP for cluster '%s'", clusterID))` for each IDP creation, earlier this was missing thats why the CI job failed  

Do you mean use different validation checkpoints (from the 4 used in this PR) for each idp creation scenario?

•```
 [FAILED] [11.749 seconds]
Edit IDP [It] can create/delete the HTPasswd IDPs - [id:49137] [feature-idp, Critical, day2]
/go/src/github.com/openshift/rosa/tests/e2e/test_rosacli_idp.go:296

  [FAILED] Expected
      <string>: INFO: Configuring IDP for cluster '2c52jne75v1ccbhk444k0ubapp9iqsoe'
      INFO: Identity Provider 'htpasswdn1' has been created.
      
  to contain substring
      <string>: To log in to the console, open
  In [It] at: /go/src/github.com/openshift/rosa/tests/e2e/test_rosacli_idp.go:334 @ 06/26/24 20:40:26.725
```

Reply:
I apologize; I did not realize they are for kinds of IDPs; it is truly required for validation.

I realized that almost all of the validation `Expectations are the same; how about collecting them into one function?